src/include/daos_errno.h

```diff
@@ -43,143 +43,209 @@ extern "C" {
 
 #define D_FOREACH_GURT_ERR(ACTION)					\
 	/** no permission */						\
-	ACTION(DER_NO_PERM,		(DER_ERR_GURT_BASE + 1))	\
+	ACTION(DER_NO_PERM,		(DER_ERR_GURT_BASE + 1),	\
+	       Operation not permitted)					\
 	/** invalid handle */						\
-	ACTION(DER_NO_HDL,		(DER_ERR_GURT_BASE + 2))	\
+	ACTION(DER_NO_HDL,		(DER_ERR_GURT_BASE + 2),	\
+	       Invalid handle)						\
 	/** invalid parameters */					\
-	ACTION(DER_INVAL,		(DER_ERR_GURT_BASE + 3))	\
+	ACTION(DER_INVAL,		(DER_ERR_GURT_BASE + 3),	\
+	       Invalid parameters)					\
 	/** entity already exists */					\
-	ACTION(DER_EXIST,		(DER_ERR_GURT_BASE + 4))	\
+	ACTION(DER_EXIST,		(DER_ERR_GURT_BASE + 4),	\
+	       Entity already exists)					\
 	/** nonexistent entity */					\
-	ACTION(DER_NONEXIST,		(DER_ERR_GURT_BASE + 5))	\
+	ACTION(DER_NONEXIST,		(DER_ERR_GURT_BASE + 5),	\
+	       The specified entity does not exist)			\
 	/** unreachable node */						\
-	ACTION(DER_UNREACH,		(DER_ERR_GURT_BASE + 6))	\
+	ACTION(DER_UNREACH,		(DER_ERR_GURT_BASE + 6),	\
+	       Unreachable node)					\
 	/** no space on storage target */				\
-	ACTION(DER_NOSPACE,		(DER_ERR_GURT_BASE + 7))	\
+	ACTION(DER_NOSPACE,		(DER_ERR_GURT_BASE + 7),	\
+	       No space on storage target)				\
 	/** already did sth */						\
-	ACTION(DER_ALREADY,		(DER_ERR_GURT_BASE + 8))	\
+	ACTION(DER_ALREADY,		(DER_ERR_GURT_BASE + 8),	\
+	       Already did sth)						\
```

Comment:
Perhaps "Action already taken" or "Operation already performed".  If nothing else, we should at least expand sth to something